lib/features/vpc/pages/give_vpc_page.dart

```diff
@@ -57,27 +57,26 @@ class GiveVPCPage extends StatelessWidget {
 }
 
 Widget _createWebViewPage(BuildContext context, VPCResponse response) {
-  final webViewController = WebViewController()
-    ..setJavaScriptMode(JavaScriptMode.unrestricted)
-    ..setNavigationDelegate(
-      NavigationDelegate(
-        onPageStarted: (String url) {
-          log('VPC onPageStarted: $url');
-          if (url == response.cancelUrl) {
-            context.read<VPCCubit>().redirectOnCancel();
-          } else if (url == response.successUrl) {
-            context.read<VPCCubit>().redirectOnSuccess();
-          }
-        },
-      ),
-    )
-    ..loadRequest(Uri.parse(response.url));
-
   return Container(
     width: double.infinity,
     height: double.infinity,
     margin: const EdgeInsets.only(top: 30),
-    child: WebViewWidget(controller: webViewController),
+    child: InAppWebView(
+      initialUrlRequest: URLRequest(url: Uri.parse(response.url)),
+      onWebViewCreated: (controller) {
+        controller.loadUrl(
+          urlRequest: URLRequest(url: Uri.parse(response.url)),
+        );
+      },
+      onLoadStart: (controller, url) {
+        log('VPC onPageStarted: $url');
```

Comment:
Maybe you use the logging service to send these logs to kibana

Reply:
do we use it already in the app? could you please guide me there?